### ISSUE TITLE
UI glitches: Pause icon incorrect, icon size differs

### ISSUE THREAD

**Issue by perlun** [ShyykoSerhiy/vscode-spotify#31]
Hi!

Thanks for an interesting extension. 👍 I am using it on macOS.

I noted two minor details:

1. The pause button is slightly "wrong", it looks like a "stop" button instead of pause: https://github.com/ShyykoSerhiy/vscode-spotify/blob/master/src/SpotifyControls.ts#L72
2. The width of `primitive-square` and `triangle-right` differs with a pixel or two, which means that the status bar "jumps around" when you play/pause the music.

Suggestions on how to fix these, and I'll gladly submit a PR.

**Comment by ShyykoSerhiy** [comment]
Yeah, vscode uses octicons(https://octicons.github.com/) for 'special' characters support (@see https://code.visualstudio.com/docs/extensionAPI/vscode-api#StatusBarItem text property). Sadly there is no 'Pause' octicon. We can try to use pause emoji for this ( https://emojipedia.org/double-vertical-bar/ ) but it will look a bit off. I don't think there is a good solution for this issue.

As for width difference we can try add space character to play or pause button (whichever is smaller).

**Comment by perlun** [comment]
Alright, we can live with the icon for now. I think the pixel error is perhaps more important.
